docs/docs/schema-customization.md

```diff
@@ -147,6 +151,7 @@ not a valid Date.
 
 This will confuse Gatsby's type inference since the `joinedAt`
 field will now have both Date and String values.
+To make it more simple in automatic type inference at times the data given to the fields may satisfy the conditions of more than one data type whihc would confuse the Graphql. Which can be further solved by the below fix using an export of a function that ensures it is a Date always.
```

Comment:
I'm not sure of the goal with this addition. The first sentence seems repetitive of the previous 2 lines and the second sentence can be inferred by the next header